packages/react/src/utils/types.ts

```diff
@@ -38,6 +38,7 @@ export type BaseProps = {
   localization?: Localization;
   tabs?: Array<Tab>;
   preferencesFilter?: PreferencesFilter;
+  containerElement?: Node | null;
```

Comment:
@LetItRock what do you think about: `container` similar to Radix terminology?

Reply:
@scopsy yes, Emotion is using similar name as well ;)

Reply:
Cool! Than let's go with it?